### DISCUSSION TITLE
구글 플레이 스토어 UGC 정책

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#138] [4. 기타 글 모음]
제가 만들고 있는 게 안드로이드 앱이다보니 구글 플레이스토어 심사를 통과하는 것도 중요한 요구사항 중 하나인데요~~!
찾아보니 커뮤니티성을 띄는 어플의 경우 규제가 빡세다고 하더라구요. 그래서 알아봤습니다~~
[게시글 링크](https://amaran-th.github.io/%EC%A3%BC%EC%A0%80%EB%A6%AC%EC%A3%BC%EC%A0%80%EB%A6%AC/[Android]%20%EA%B5%AC%EA%B8%80%20%ED%94%8C%EB%A0%88%EC%9D%B4%EC%8A%A4%ED%86%A0%EC%96%B4%20UGC%20%EC%A0%95%EC%B1%85/)

**Comment by pnudpwls1103** [comment]
오.. 이런 정책이 있는지 처음 알았네요! 게임 쪽은 빌드할 때마다 심사를 거쳐야하는데 그 기간이 길어서 요행으로 심사를 피하는 방법이 있다고 하더라구요. 오늘도 유용한 포스팅 곰마워요!!
